### DISCUSSION TITLE
It is suggested that the future development direction is still based on full memory, and supports K33 and K34.

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/bladebit#233] [Ideas]
come on

**Comment by NTOHPN** [comment]
Does anyone care?

**Comment by harold-b** [comment]
For the time being, it isn't currently planned

**Comment by ejko** [comment]
It will be nice to plot K33 size.

**Comment by hajes** [comment]
I would be more interested in decompression optimisation. Computing overhead, plot size, lag significantly behind Gigahorse/MMX.

k33 would on my second wish list.
